Cargo.toml

```diff
@@ -40,7 +40,7 @@ lru = "0.12"
 near-crypto = { git = "https://github.com/near/nearcore", tag = "1.38.2" }
 near-indexer = { git = "https://github.com/near/nearcore", tag = "1.38.2" }
 near-primitives = { git = "https://github.com/near/nearcore", tag = "1.38.2" }
-near-lake-framework = "0.7"
+near-lake-framework = { git = "https://github.com/aurora-is-near/near-lake-framework-rs", branch = "0.7.x" }
```

Comment:
I would use hash commit here to prevent the appearance of potential breaking changes in the branch in the future.